src/platform-implementation-js/dom-driver/gmail/gmail-driver/nav-item-section.ts

```diff
@@ -0,0 +1,122 @@
+import autoHtml from 'auto-html';
+import querySelector from '../../../lib/dom/querySelectorOrFail';
+import { panelNavItemsContainerSelector } from '../../../views/collapsible-panel-view';
+import insertElementInOrder from '../../../lib/dom/insert-element-in-order';
+
+export const getSectionClassName = (sectionKey: string) =>
+  `inboxsdk__navItem_section_${sectionKey}` as const;
+
+const getSectionListClassName = (sectionKey: string) =>
+  `inboxsdk__navItem_section_${sectionKey}_list` as const;
```

Comment:
Instead of CSS :globals, I wonder if custom data attributes might be a lot cleaner here.

`inboxsdk__navItem_section_${sectionKey}_list` could be deconstructed with a 'list' and the key as separate data attributes.

Instead of a :global, we'd add a CSS :local. The sectionKey wouldn't have to be baked into each individual global class name, just at the top of its particular tree. Something like `Element.dataset.inboxSdkSectionKey = sectionKey`.